docs/Wallets/Core-Wallet.md

```diff
@@ -70,19 +70,23 @@ You then can launch Vertcoin via the app icon installed  in the launcher.
 
 ##  Running Vertcoin Core
 
-The first time you run Vertcoin Core, it wil ask where you would like to store its data. You may chose to store data in the default directory or you may chose your own. After selecting a directory and clicking next, Vertcoin Core will attempt to connect to other Vertcoin Core nodes and download a complete copy of the blockchain.
+The first time you run Vertcoin Core, it will ask where you would like to store its data. You may choose to store data in the default directory or you may choose your own. After selecting a directory and clicking next, Vertcoin Core will attempt to connect to other Vertcoin Core nodes and download a complete copy of the blockchain.
 
 
 !!!caution "Note"
-    The Vertcoin Core Wallet requires at least 2 GB of space on your computer to store the blockchain. Downloading the entire chain for the for time may take a while.
+    The Vertcoin Core Wallet requires at least 2 GB of space on your computer to store the blockchain. Downloading the entire chain may take a while.
 
-As the blockchain downloads, you'll see progress displayed as well as the latest block synced. Every time Vertcoin Core is launced, your local copy of the blockchain will need to be synced to the current block.
+As the blockchain downloads, you'll see progress displayed as well as the latest block synced. Every time Vertcoin Core is launched, your local copy of the blockchain will need to be synced to the current block.
 
 
 
 ## Backing Up Your Wallet
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque viverra, sapien nec feugiat ultrices, tortor tellus euismod quam, vel luctus ipsum dolor eu quam. Duis lacinia consequat sapien, rutrum dignissim neque faucibus vitae.
+To backup your wallet
+
+
+!!!caution "Note"
+    Restoring your Vertcoin Core Wallet requires the core wallet.dat file.  There are no seed words.
 
 ## Sending & Recieving Vertcoin
 Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque viverra, sapien nec feugiat ultrices, tortor tellus euismod quam, vel luctus ipsum dolor eu quam. Duis lacinia consequat sapien, rutrum dignissim neque faucibus vitae.
```

Comment:
If you are still working on a section, either tag it as a "WIP" (Work in Progress) or don't include it and raise an issue to say that there's something missing and you're working on it.